src/scenekit.cs

```diff
@@ -4306,4 +4348,121 @@ interface SCNBufferStream {
+	
+		[Static, NoWatch]
+		[Export ("functionWithCAMediaTimingFunction:")]
+		SCNTimingFunction Create (CAMediaTimingFunction caTimingFunction);
+	}
+
+	[Watch (4,0), TV (11,0), Mac (10,13), iOS (11,0)]
+	[BaseType (typeof(NSObject))]
+	interface SCNAnimation : NSCopying, NSSecureCoding
+	{
+		[Static]
+		[Export ("animationWithContentsOfURL:")]
+		SCNAnimation FromUrl (NSUrl animationUrl);
+	
+		[Static]
+		[Export ("animationNamed:")]
+		SCNAnimation FromName (string animationName);
+	
+		[Static, NoWatch]
+		[Export ("animationWithCAAnimation:")]
+		SCNAnimation FromCAAnimation (CAAnimation caAnimation);
+	
+		[Export ("duration")]
+		double Duration { get; set; }
+	
+		[NullAllowed, Export ("keyPath")]
+		string KeyPath { get; set; }
+	
+		[Export ("timingFunction", ArgumentSemantic.Retain)]
+		SCNTimingFunction TimingFunction { get; set; }
+	
+		[Export ("blendInDuration")]
+		double BlendInDuration { get; set; }
+	
+		[Export ("blendOutDuration")]
+		double BlendOutDuration { get; set; }
+	
+		[Export ("removedOnCompletion")]
+		bool RemovedOnCompletion { [Bind ("isRemovedOnCompletion")] get; set; }
+	
+		[Export ("appliedOnCompletion")]
+		bool AppliedOnCompletion { [Bind ("isAppliedOnCompletion")] get; set; }
+	
+		[Export ("repeatCount")]
+		nfloat RepeatCount { get; set; }
+	
+		[Export ("autoreverses")]
+		bool Autoreverses { get; set; }
+	
+		[Export ("startDelay")]
+		double StartDelay { get; set; }
+	
+		[Export ("timeOffset")]
+		double TimeOffset { get; set; }
+	
+		[Export ("fillsForward")]
+		bool FillsForward { get; set; }
+	
+		[Export ("fillsBackward")]
+		bool FillsBackward { get; set; }
+	
+		[Export ("usesSceneTimeBase")]
+		bool UsesSceneTimeBase { get; set; }
+	
+		[NullAllowed, Export ("animationDidStart", ArgumentSemantic.Copy)]
+		SCNAnimationDidStartBlock AnimationDidStart { get; set; }
+	
+		[NullAllowed, Export ("animationDidStop", ArgumentSemantic.Copy)]
+		SCNAnimationDidStopBlock AnimationDidStop { get; set; }
+	
+		[NullAllowed, Export ("animationEvents", ArgumentSemantic.Copy), NoWatch]
+		SCNAnimationEvent[] AnimationEvents { get; set; }
+	
+		[Export ("additive")]
+		bool Additive { [Bind ("isAdditive")] get; set; }
+	
+		[Export ("cumulative")]
+		bool Cumulative { [Bind ("isCumulative")] get; set; }
+	}
+	
+	[Watch (4,0), TV (11,0), Mac (10,13), iOS (11,0)]
+	[BaseType (typeof(NSObject))]
+	interface SCNAnimationPlayer : SCNAnimatable, NSCopying, NSSecureCoding
+	{
+		[Static]
+		[Export ("animationPlayerWithAnimation:")]
+		SCNAnimationPlayer FromAnimation (SCNAnimation animation);
+	
+		[Export ("animation")]
+		SCNAnimation Animation { get; }
+	
+		[Export ("speed")]
+		nfloat Speed { get; set; }
+	
+		[Export ("blendFactor")]
+		nfloat BlendFactor { get; set; }
+	
+		[Export ("paused")]
+		bool Paused { get; set; }
+	
+		[Export ("play")]
+		void Play ();
+	
+		[Export ("stop")]
+		void Stop ();
+	
+		[Export ("stopWithBlendOutDuration:")]
+		void StopWithBlendOutDuration (double duration);
```

Comment:
I think we can just use just `Stop` here and use `blendOutDuration` as parameter name instead